docs/environment.yml

```diff
@@ -2,7 +2,8 @@ name: readthedocs
 channels:
   - conda-forge
 dependencies:
+  - linkchecker=10.4.*
   - python=3.12
-  - sphinx_rtd_theme=2.0.*
+  - sphinx_rtd_theme=3.0.*
```

Comment:
Will this update fix our need to pin Python? We had a bit of a problem with that while you were away. We might want to consider removing that line of this file?

It may also be nicer for us to pin it given that it gave us a bit of a hard time in the middle of trying to solve a problem for uwtools quickly.

Reply:
Interesting. I think I recall seeing the exchange about Python 3.13 and docs generation when I was on vacation. Let me try a couple things here before responding.